framework/db/Connection.php

```diff
@@ -618,9 +624,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
+        $command = Yii::createObject([
```

Comment:
Hm... Didn't thought about this. It will be really faster. Since it's configurable with class options, it makes not much sense to change class with DI